### ISSUE TITLE
Idee RKMP

### ISSUE THREAD

**Issue by Kensiee** [Rammelkast/RKMP-Premium-v2#2]
Ik ben zeker een RKMP klant
Ik weet dat het misschien moeilijk is om te maken maar het is misschien wel nice om MySQL saving in de plugin te krijgen en dus ook een soort van telefoon waar je global berichten tussen servers in de bungee kunt mee sturen :)

**Comment by Rammelkast** [comment]
SQLite support is standaard, hierdoor kan ik makkelijk MySQL support toevoegen.
Kortom: dit gaat er zeker komen.

**Comment by Kensiee** [comment]
Misschien zou het ook wel nice zijn om het adminmenu in de game te hebben :)

**Comment by Jamie4224** [comment]
Ja, dit zou inderdaad moeten worden toegevoegd, misschien zijn er wel zo grote minetopia servers die regio's per server hebben en dit nodig hebben! 

Toevoegingen aan dit idee:
- Autosynchronisatie op tijd interval!
- Flexibele database style. Makkelijk te integreren in bv. websites.

  - Jamie
